src/classiceditorui.js

```diff
@@ -18,24 +19,62 @@ import normalizeToolbarConfig from '@ckeditor/ckeditor5-ui/src/toolbar/normalize
  */
 export default class ClassicEditorUI extends EditorUI {
 	/**
-	 * @inheritDoc
+	 * Creates an instance of the classic editor UI class.
+	 *
+	 * @param {module:core/editor/editor~Editor} editor The editor instance.
+	 * @param {module:ui/editorui/editoruiview~EditorUIView} view The view of the UI.
 	 */
 	constructor( editor, view ) {
```

Comment:
I think there is no point to pass view as a parameter here now. Is should be created inside this class.

Reply:
cc @Reinmar

Reply:
Nah, AFAIR it was meant that it's `Editor` that glues `EditorUI` and `EditorUIView` together because it makes it easy to customize things. If we hardcode the `EditorUIView` inside `EditorUI`, one must re–write `EditorUI` from scratch to customize the view.

Reply:
Except for the fact that it works with `ClassicEditor` this way and other editor creators create `*UIView` using `sourceElement`'s. So the entire `view` instance needs to be passed (the way it is now) or a `sourceElement` (but then it will be somehow inconsistent with parent class which had, now deprecated, `view` param).

Reply:
Ok, let's keep it the way it is.

Reply:
👍